app/email.go

```diff
@@ -101,8 +101,8 @@ func SendVerifyEmail(userId, userEmail, locale, siteURL string) *model.AppError
 func SendSignInChangeEmail(email, method, locale, siteURL string) *model.AppError {
 	T := utils.GetUserTranslations(locale)
 
-	subject := T("api.templates.singin_change_email.subject",
-		map[string]interface{}{"SiteName": utils.ClientCfg["SiteName"]})
+	subject := fmt.Sprintf("[%v] %v", utils.Cfg.TeamSettings.SiteName, T("api.templates.singin_change_email.subject",
```

Comment:
String formatting should be done in the loc stuff?  No @enahum 

like

```go
api.templates.singin_change_email.subject := "[{{ .SiteName }}] You updated your sign-in method on {{ .SiteName }}"

subject := T("api.templates.singin_change_email.subject", map[string]interface{}{"SiteName": utils.ClientCfg["SiteName"]}))
```

What happens when we want to change `[]` to `()` or `[]` has a special meaning in a different language?